packages-ui/roosterjs-react/lib/emoji/type/AriaAttributes.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @internal
+ */
+export const AriaAttributes = {
```

Comment:
Use const enum instead

Reply:
And this is only used in createEmojiPlugin.ts, so you can move it into that file and no need to export